TangoRosStreamer/app/build.gradle

```diff
@@ -27,7 +27,7 @@ dependencies {
     compile 'org.ros.rosjava_messages:sensor_msgs:[0.3,)'
     compile 'org.ros.rosjava_messages:geometry_msgs:[0.3,)'
     compile 'org.ros.rosjava_messages:dynamic_reconfigure:[0.3,)'
-    compile 'org.ros.rosjava_messages:tango_ros_messages:[0.0,)'
+    compile 'org.ros.rosjava_messages:tango_ros_messages:1.0.0'
```

Comment:
Suggestion: use [1.0.0, 1.1) format to support minor changes that don't break the API just in case.